### PR TITLE
Create za-sars.json closes #448

### LINKED ISSUE #448: PROPOSAL: New South African public university registrar
I am trying to find a suitable registrar for a South African public University. They are not registered as a nonprofit, thus [ZA-NPO](http://org-id.guide/list/ZA-NPO) can't be used for them.

I did some research on other options, I found that the Department of Higher Education and Training (DHET) does  the registration and accreditation status of universities, however, they do not issue a registration number for public universities only for private ones. Here is an extract from an accreditation letter sent by a university:
**In terms of the Higher Education Act (Act 101 of 1997) and the SU Statute (Government Notice No. 42636 Notice  1062  of  16  August  2019)  Stellenbosch  University  is  a  public  higher  education  institution.  Please  note that, in terms of the Act, only private providers are given registration numbers.**

Looking at other universities accreditation statuses as well seems like the government notice number and date is what is considered unique for a university. Cape Town uni has: **Government Notice No. 1199 of 20 September 2002.**
http://www.rci.uct.ac.za/rcips/contracts/admin_info

Could I use this number as a registration number? For example for Stellenbosch  University we could use _42636-1062-16082019_

Alternatively, they have an Income Tax Number issued by the [South African Revenue Service (SARS)](https://www.sars.gov.za/businesses-and-employers/ ). Ex. 9045478147

Which new registrar would be preferred for universities, DHET or SARS?

### PR DESCRIPTION
**Pull request by andreaszenasidi**
Github issue https://github.com/org-id/register/issues/448

### CODE REVIEW

**Comment by kd-ods** [comment]
@andreaszenasidi - you say in the description 'Businesses can receive multiple tax numbers such as CIT, VAT, PAYE etc'. Since that is the case, the guidance on locating (and recording IDs) isn't quite enough ("Ask the organisation you wish to identify to provide it's tax number"). We will need to suggest that the type of tax number is prefixed in a standard way onto the tax number, in order to create a unique id. I'm just looking for examples of where we've done this elsewhere, to give you a helping hand.

**Comment by kd-ods** [comment]
Actually, @andreaszenasidi - I think we should follow the pattern set by @siwhitehouse when he added separate [org-id entries](http://org-id.guide/results?structure=all&coverage=KG&sector=all) for lists managed by the Ministry of Justice in the Kyrgyz Republic.

Practically, that means turning this entry into za-cit, I think. (Or whichever identifier it is that your publisher is going to use.) And making clear that the SARS is the issuer/manager of these IDs.

How does that sound?

**Comment by andreaszenasidi** [comment]
@kd-ods thanks for the suggestion. I renamed the JSON to **za-cit** and made the necessary changes in the JSON itself as you requested. I hope it's good now.

File: lists/za/za-cit.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  "sector": [],
+  "code": "ZA-CIT",
+  "confirmed": true,
+  "deprecated": false,
+  "listType": "secondary",
+  "access": {
+    "availableOnline": false,
+    "onlineAccessDetails": null,
+    "publicDatabase": "http://",
+    "guidanceOnLocatingIds": "Ask the organisation you wish to identify to provide it's Corporate Income Tax number, sometimes referred to as Income Tax number.",
```
Comment (kd-ods) [inline lists/za/za-cit.json:29]: spelling: "... provide its Corporate..."

**Review by kd-ods — CHANGES_REQUESTED** [review]
Couple of minor edits requested.

**Comment by andreaszenasidi** [comment]
@kd-ods I made all requested changes.

**Review by kd-ods — APPROVED** [review]

**Pull request merged**